self-service/forgerock-selfservice-core/src/main/java/org/forgerock/selfservice/core/config/ClassNameFallbackPropertyTypeResolver.java

```diff
@@ -50,9 +52,8 @@
     @Override
     public TypeDeserializer buildTypeDeserializer(
             final DeserializationConfig config, final JavaType baseType, final Collection<NamedType> subtypes) {
-
         // important to get the normal TypeIdResolver!
-        final TypeIdResolver idRes = this.idResolver(config, baseType, subtypes, false, true);
+        final TypeIdResolver idRes = this.idResolver(config, baseType,verifyBaseTypeValidity(config, baseType), subtypes, false, true);
```

Comment:
Missing space... also not sure what is this change about.

Reply:
Added the missing space. 

Change comments:
StdTypeResolverBuilder::idResolver requires a new argument: PolymorphicSubtypeValidator, (This handles prohibition of deserialization of gadgets considered insecure). 

Newly added call constructs this validator the same way superclass does. It was taken from `StdTypeResolverBuilder::buildTypeDeserializer`